vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -4,9 +4,24 @@
 import torch
 import torch.jit
 
+from vllm.logger import init_logger
 from vllm.model_executor.layers.spec_decode_base_sampler import (
     SpecDecodeStochasticBaseSampler)
 
+logger = init_logger(__name__)
+
+try:
+    """
+    Consider utilizing the FlashInfer rejection sampling kernel initially,
+    as it employs a dedicated kernel rather than relying on 
+    Torch tensor operations. This design choice helps to fuse operations, 
+    reduce memory I/O, and consequently enhances performance.
+    """
+    from flashinfer.sampling import chain_speculative_sampling
+    logger.info("Use flashinfer for rejection sampling.")
```

Comment:
nit: also log in the case when flashinfer is not used (again, makes debugging easier)